### ISSUE TITLE
Disappearing toolbar buttons

### ISSUE THREAD

**Issue by jasongfleischer** [jupyterlab/jupyterlab#3331]
Jupyterlab 0.27.0 running via Anaconda in Safari on MacOS 10.12.6, using a python 2.7 notebook.

After a couple of hours of operation, Jupyterlab's notebook toolbar buttons (Save, Cut, Paste, etc. ) become blank.

Even though the buttons are blank, tooltips still pop up when you hover over, and you can press the buttons.  The buttons return if I press reload in the browser.

The copy/cut/paste buttons still work, but the save button stops working.

I repeat, the blank save button will press, but it doesn't save.   It's pretty disheartening to loose a few hours of work this way.

This has happened to me three times, so it seems like its truly a thing, and not a random occurrence.

Thanks for your efforts in developing Jupyterlab!

**Comment by blink1073** [comment]
I'm guessing this is a Safari-specific bug that has hopefully been fixed in newer releases of Safari.

**Comment by NicWayand** [comment]
Yes, this just started happening to me as well. 

jupyterlab     0.32.1
Chrome Version 67.0.3396.99 (Official Build) (64-bit)

![image](https://user-images.githubusercontent.com/1117224/42058420-3cc0a29e-7ad5-11e8-9ec8-f6190bfe3f6e.png)

**Comment by jasongrout** [comment]
Thanks. Do you see any errors in the Javascript console? What operating system are you on? Do you notice it happening in other browsers?

I haven't seen this ever happen, so whatever help you can give to try to reproduce it would be very helpful.

**Comment by NicWayand** [comment]
Yes, see below.
Linux version 4.14.12 (gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1))
I did not see it happening in firefox.
Thanks!

```bash
[W 14:08:09.526 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 3.58ms referer=http://localhost:8889/lab
[W 14:08:26.205 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 3.37ms referer=http://localhost:8889/lab
[W 14:08:29.818 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 2.89ms referer=http://localhost:8889/lab
[W 14:08:43.307 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 2.44ms referer=http://localhost:8889/lab
[W 14:08:44.976 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 1.70ms referer=http://localhost:8889/lab
[W 14:08:57.211 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/circle.svg (127.0.0.1) 2.15ms referer=http://localhost:8889/lab
[W 14:09:10.437 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/stop.svg (127.0.0.1) 2.53ms referer=http://localhost:8889/lab
[W 14:09:10.450 LabApp] 404 GET /lab/api/themes/@jupyterlab/theme-light-extension/icons/md/stop.svg (127.0.0.1) 1.96ms referer=http://localhost:8889/lab
```

**Comment by jasongrout** [comment]
Those errors match well with the symptoms - it can't find the svg files for the buttons, apparently.

Did you install any extensions, or are you using the stock jupyterlab? Does it still happen with `jupyter lab --core-mode`?

**Comment by ian-r-rose** [comment]
Now that we webpack the themes, those icons should no longer be located on the server: rather they should be delivered with the whole theme bundle. Seems like there is something out of sync with your install. Perhaps `jupyterlab_launcher` needs to be updated?

**Comment by saulshanabrook** [comment]
Also could you try force reloading to invalidate the cache? Some people
have had trouble when they pull in an old theme file.

**Comment by NicWayand** [comment]
It is fixed now, I think I had an older jupyterlab instance running while I updated via conda.

**Comment by ian-r-rose** [comment]
I think that since the original issue was likely solved by webpacking the themes, we can safely close this as fixed. @jasongfleischer, if you still see this on the most recent version of JupyterLab, please let us know and we can reopen.

**Comment by hdavid16** [comment]
I have just installed a clean version of python using miniconda. I installed jupyterlab as instructed in the documentation: conda install -c conda-forge jupyterlab. It installed version 0.35.0. However, the save button is disabled on any notebook I open or create.

I should note that I was getting a No Kernel Error at first and had to run: python -m ipykernel install --user. Other than that, everything else is stock.

Any thoughts?
